sympy/logic/tests/test_inference.py

```diff
@@ -388,6 +388,3 @@ def make_random_cnf(num_clauses=5, num_constraints=10, num_var=2):
         assert z3_sat == lra_dpll2_sat
         if z3_sat:
             sat_count += 1
-
-    # print("lra_dpll2_time", lra_dpll2_time)
-    # print("z3_time", z3_time)
```

Comment:
Also remove the timing stuff from the test. If we want to keep track of timings we should do that in the benchmarking suite.